subprojects/instant-execution/src/main/kotlin/org/gradle/instantexecution/DefaultInstantExecution.kt

```diff
@@ -481,7 +481,7 @@ class DefaultInstantExecution internal constructor(
 
     private
     val instantExecutionStateFile by unsafeLazy {
-        val cacheDir = absoluteFile(".instant-execution-state/${currentGradleVersion()}")
+        val cacheDir = absoluteFile(".gradle/${currentGradleVersion()}/configuration-cache")
```

Comment:
Maybe flip these around, so that we use `.gradle/configuration-cache/version`, as this will make it easier for people to find and delete the cache if they need to. Or we could look at including the Gradle version in the cache key hash and remove the version from the cache directory path.

Reply:
👍 for `./gradle/configuration-cache/$gradleVersion`